app/templates/account/settings.hbs

```diff
@@ -24,8 +24,8 @@
   {{#if scrollToEmail}}
     {{scroll-here}}
   {{/if}}
-  <section class="email-settings" data-test-email-settings>
-    <h3>Email notifications</h3>
+  <section id="email-settings" class="email-settings" data-test-email-settings>
+    <h3 id="email-notifications">Email notifications</h3>
```

Comment:
Why do sections need `id`? I didn't notice any uses of them. Generally, I would avoid using `id`s, as it's hard to track their uniqueness on the page with dynamic views. And also Ember sets `id`s for some nodes for it's own purposes.

Reply:
@MariadeAnton requested that we be able to give out `#` links to specific sections, but yeah with the dynamic nature of the views maybe it is precarious. Maybe there's a better way to achieve the same thing?

Reply:
We have a `scroll-here` component for such purposes. Here's an example: https://github.com/travis-ci/travis-web/blob/master/app/templates/help.hbs#L1

Basically, first you define anchor query parameters in controller:
```js
export const SECTION = {
  EMAIL_SETTINGS: 'email-settings',
  INSIGHTS: 'insights'
}

export default Constroller.extend({
  queryParams: ['section'],
  section: null,
  
  isEmailSettingsSection: equal('section', SECTION.EMAIL_SETTINGS),
  isInsightsSection: equal('section', SECTION.INSIGHTS)
})
```

And then, depending on query parameter, you can define the places to scroll to in the template:
```hbs
{{#if isEmailSettingsSection}}{{scroll-here}}{{/if}}
...
{{#if isInsightsSection}}{{scroll-here}}{{/if}}
```

Reply:
Nice! I'll switch to that.

Reply:
Updated to use `section` query parameter and `scroll-here` 😄